### DISCUSSION TITLE
Max brightness at 50% of slider

### DISCUSSION THREAD

**Discussion by JuliusNmn** [MonitorControl/MonitorControl#1470] [General]
### Before opening the issue, have you...?

- [x] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Brightness Range seems to be incorrect. My monitor reaches max brightness at 50% of the slider range.

### Expected behavior

50% of the slider should get me 50% brightness.

### Environment Information (please complete the following information)

```markdown
- macOS version: 14.0 (23A344)
- Mac model: MacBook Pro 14-inch, 2021
- MonitorControl version: 4.2.0
- Monitor(s): Samsung U32J59x
- Monitor Cable(s)/Connection(s): USB hub
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Hi, it seems like the connection or the display does not support hardware brightness control which the app uses by default in the 50-100% range. Try disabling DDC for this display so the app uses software dimming for the entire range.

**Reply by JuliusNmn** [reply]
that was the problem. thanks!
